app/os_js.go

```diff
@@ -162,6 +176,24 @@ func (w *window) cleanup() {
 }
 
 func (w *window) addEventListeners() {
+	w.addEventListener(w.cnv, "webglcontextlost", func(this js.Value, args []js.Value) interface{} {
+		args[0].Call("preventDefault")
+		w.contextStatus = contextStatusLost
+		w.SetAnimating(false)
```

Comment:
app.Window is in control of SetAnimating; it seems inappropriate to force it here.